scripts/inject-headless-version.js

```diff
@@ -0,0 +1,36 @@
+const {readFileSync, writeFileSync} = require('fs');
+
+const packages = ['angular', 'cra-template', 'vue-cli-plugin-typescript'];
+const headlessVersion = '0.x';
```

Comment:
This will be changed I presume by `^1.0.0` when the v1 will be released?

Reply:
Yes, exactly.